# yolov8n.yaml

yolo_model:
  # PyTorch 모델의 경로 설정
  weight_file:
    name: /home/nick/lee/src/darknet_ros_d435i/darknet_ros/yolo_network_config/weights/yolov8n.pt  # PyTorch 모델 파일(.pt)
  
  # YOLOv8 모델 하이퍼파라미터
  threshold:
    value: 0.3  # 탐지 임계값 설정 (필요에 따라 조정 가능)
  detection_classes:
    names:
      - person
      - bicycle
      - car
      - motorcycle
      - airplane
      - bus
      - train
      - truck
      - boat
      - traffic light
      - fire hydrant
      - stop sign
      - parking meter
      - bench
      - bird
      - cat
      - dog
      - horse
      - sheep
      - cow
      - elephant
      - bear
      - zebra
      - giraffe
      - backpack
      - umbrella
      - handbag
      - tie
      - suitcase
      - frisbee
      - skis
      - snowboard
      - sports ball
      - kite
      - baseball bat
      - baseball glove
      - skateboard
      - surfboard
      - tennis racket
      - bottle
      - wine glass
      - cup
      - fork
      - knife
      - spoon
      - bowl
      - banana
      - apple
      - sandwich
      - orange
      - broccoli
      - carrot
      - hot dog
      - pizza
      - donut
      - cake
      - chair
      - couch
      - potted plant
      - bed
      - dining table
      - toilet
      - TV
      - laptop
      - mouse
      - remote
      - keyboard
      - cell phone
      - microwave
      - oven
      - toaster
      - sink
      - refrigerator
      - book
      - clock
      - vase
      - scissors
      - teddy bear
      - hair drier
      - toothbrush

model_parameters:
  imgsz: 640  # 입력 이미지 크기
  device: 'cpu'  # 또는 'cuda' (GPU 사용 시)
